### PR TITLE
tp-qemu: qemu_io_blkdebug: no need to start vm for the case.

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1248408

### CODE REVIEW

**Comment by xiagao** [comment]
Acked-by <xiagao@redhat.com>

File: qemu/tests/cfg/qemu_io_blkdebug.cfg [hunk 1]
```diff
@@ -6,6 +6,7 @@
     image_name_blk = "images/blkdebug"
     image_size_blk = 20G
     image_format_blk = qcow2
+    force_create_image_blk = yes
```
Comment (xiagao) [inline qemu/tests/cfg/qemu_io_blkdebug.cfg]: Is this should be "force_create_image =  yes" ?

**Comment by xutian** [comment]
LGTM

**Comment by xiagao** [comment]
Acked-by: <xiagao@redhat.com>

**Comment by XueqiangWei** [comment]
LGTM,
Acked by Xueqiang Wei <xuwei@redhat.com>

**Pull request merged**